spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe API::V1::UsersController do
```

Comment:
Oh yea, completely missed this. The API is tested with request tests, eg. `spec/requests/api/v1/users_spec.rb`.